smarts/core/scenario.py

```diff
@@ -454,74 +454,67 @@ def _discover_bubbles(scenario_root):
     def set_ego_missions(self, ego_missions: dict):
         self._missions = ego_missions
 
+    def get_vehicle_start_at_time(
+        vehicle_id: str, start_time: float
+    ) -> Tuple[Start, float]:
```

Comment:
I could suggest a docstring and an entry in the `CHANGELOG`.

Reply:
done.